internal/keycloakb/usersdbmodule_test.go

```diff
@@ -108,6 +182,24 @@ func TestGetUserInformation(t *testing.T) {
 		assert.Equal(t, unexpectedError, err)
 	})
 
+	t.Run("Error at decryption", func(t *testing.T) {
+		var natureValue = "nature"
+		gomock.InOrder(
+			mockDB.EXPECT().Query(gomock.Any(), realm, userID).Return(mockSQLRows, nil),
+			mockSQLRows.EXPECT().Next().Return(true),
+			mockSQLRows.EXPECT().Scan(gomock.Any()).DoAndReturn(func(checkID *int64, realm *string, userID *string, operator *sql.NullString,
+				datetime *sql.NullString, status *sql.NullString, checkType *sql.NullString, nature *sql.NullString, proofType *sql.NullString,
+				proofData *[]byte, comment *sql.NullString) error {
```

Comment:
You can do it simpler:
* for each unused parameter, you can use _ rather than naming it and user interface{}
Then, you can simplify the writing interface{} parameters:
* `_ interface{}, _ interface{}, _ interface{}` becomes `_, _, _ interface{}`

You finally get something like:
`mockSQLRows.EXPECT().Scan(gomock.Any()).DoAndReturn(func(_, _, _, _, _, _, _ interface{}, nature *sql.NullString, _, _, _ interface{}) error {`

Reply:
OK